src/main/java/seedu/contax/logic/parser/FindAppointmentCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.contax.logic.parser;
+
+import static seedu.contax.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_SEARCH_TYPE;
+
+import java.util.Arrays;
+
+import seedu.contax.logic.commands.FindAppointmentCommand;
+import seedu.contax.logic.parser.exceptions.ParseException;
+import seedu.contax.model.appointment.NameContainsKeywordsPredicate;
+import seedu.contax.model.appointment.PersonNameContainsKeywordsPredicate;
+
+
+
+
+/**
+ * Parses input arguments and creates a new FindAppointment object
+ */
+public class FindAppointmentCommandParser implements Parser<FindAppointmentCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindAppointment
+     * and returns a FindAppointment object for execution.
```

Comment:
Similarly, the class names are wrong.
```suggestion
     * Parses the given {@code String} of arguments in the context of the FindAppointmentCommand
     * and returns a FindAppointmentCommand object for execution.
```

Reply:
Fixed